app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class PostsController < BaseController
+      def index
+        user = User.find(params[:user_id])
```

Comment:
Let's add an error response in case the user_id given doesn't exist in our database

Reply:
Once you add this, remember to test it in your spec as well!